README.md

```diff
@@ -38,24 +36,28 @@ Event specific query parameters should be appended to the URL as follows:
 * `event=load` – any network request
 * `time=xxx` - the time (in ms) for the network request to complete
 
-```bash
-curl -G https://raw.githubusercontent.com/bbc/news-apps-ios-coding-challenge/master/analytics?event=load&data=100
+```
+https://raw.githubusercontent.com/bbc/news-apps-ios-coding-challenge/master/analytics?event=load&data=100
 ```
 
-### Tap/Pull To Refresh
-Ideally, the user should also be able to trigger a reload of the data from the server.
+### Pull To Refresh
+Ideally, the user should also be able to trigger a reload of the data from the server, using a pull to refresh action.
 
 ### Languages
-For iOS roles, you can write the app in either Swift or Objective-C, but _you should avoid third party libraries_. If you use them, please explain why. For clarity, anything in the Apple SDKs we would consider first-party and an essential part of most any iOS app.
+For iOS roles, you can write the app in either Swift or Objective-C.
 
-For Android roles, you can write the app in either Kotlin or Java, but _you should avoid third party libraries_.  If you use them, please explain why. For clarity, anything in the Android SDKs we would consider first-party and an essential part of most any Android app.
+For Android roles, Kotlin is preferred but Java is also acceptable. 
```

Comment:
I would suggest we should remove Java - as it's less relevant these days. Kotlin is a must for new developers, would we not agree?